baseplate/context/memcache/lib.py

```diff
@@ -0,0 +1,111 @@
+import logging
+import zlib
+
+from ..._compat import (
+    long,
+    pickle,
+    string_types,
+)
+
+
+"""Memcache serialization/deserialization (and compression) helper methods.
+
+Memcached can only store strings, so to store arbitrary objects we need to
+serialize them to strings and be able to deserialize them back to their
+original form. General purpose serialization and deserialization can be
+achieved with pickle_and_compress() and decompress_and_unpickle().
+
+"""
+
+
+FLAG_PICKLE = 1 << 0
```

Comment:
Sorry, misspoke, it's just `import enum` either way but the backwords-compatibility package `enum34` will be present for py2.

https://github.com/reddit/baseplate/blob/master/baseplate/events/queue.py#L24